test/thetest.proto

```diff
@@ -583,10 +583,10 @@ message UnrecognizedWithEmbed {
 }
 
 //begin versioned: only for protoc 2.6.0 and up
-message SampleOneOf {
-  oneof test_oneof {
-    string name = 4;
-    NinOptNative sub_message = 9;
-  }
-}
+// // message SampleOneOf {
```

Comment:
I have made a fix for this that I haven't pushed yet.
My bad.
If possible could you please revert the change on this file.
Sorry.